manifests/cluster-manager/hub/cluster-manager-registration-clusterrole.yaml

```diff
@@ -42,6 +42,10 @@ rules:
 - apiGroups: ["work.open-cluster-management.io"]
   resources: ["manifestworks"]
   verbs: ["get", "list", "watch"]
+# Allow hub to monitor manifestworkreplicasets
+- apiGroups: ["work.open-cluster-management.io"]
```

Comment:
you also need manifestworkreplicasets/status, and manifestworkreplicasets/finalize permission I think

Reply:
done